src/Unused/Cache.hs

```diff
@@ -10,23 +10,26 @@ import Data.Vector (toList)
 import qualified Data.ByteString.Lazy as BS
 import Unused.Cache.DirectoryFingerprint (sha)
 
+newtype CacheFileName = CacheFileName String
```

Comment:
@tonyd256 yes, it would - however, as an alias the compiler isn't really doing much more and it's just a visual change. I'd prefer a bit more rigidity here, even in what seems like an unimportant case like this.